zeta-chain/node
# fix(hotfix): check for deactivated methods

To test:

Run smoke test
```
cd contrib/localnet
docker-compose up -d
```

Install cast: https://book.getfoundry.sh/getting-started/installation

Test one of the method:
```
cast send --private-key d87baf7bf6dc560a252596678c12e41f7d1682837f05b29d411bc3f78ae2c263 0xd97B1de3619ed2c6BEb3860147E30cA8A7dC9891 "increaseAllowance(address,uint256)" "0x3c44cdddb6a900fa2b585dd299e03d12fa4293bc" "10000" --rpc-url http://0.0.0.0:9545
```

Check allowance is not increased:
```
cast call 0xd97B1de3619ed2c6BEb3860147E30cA8A7dC9891 "allowance(address, address)" 0x3c44cdddb6a900fa2b585dd299e03d12fa4293bc 0xE5C5367B8224807Ac2207d350E60e1b6F27a7ecC --rpc-url http://0.0.0.0:9545
```
